frontend/src/styles/_uswds-theme.scss

```diff
@@ -19,6 +19,13 @@ in the form $setting: value,
   $utilities-use-important: true,
 
   // Prettier defaults for typography
-  $theme-font-type-sans: "public-sans",
-  $theme-global-link-styles: true
+  $theme-font-type-sans: "source-sans-pro",
```

Comment:
We don't have to change this; it wasn't an intentional design choice. I didn't realize that the [Truss USWDS Figma](https://www.figma.com/community/file/836611771720754351) assets I copied the components from used this font instead of what USWDS defaults to.